### ISSUE TITLE
Make addresses clickable and have them open the address in Google Maps in a new tab

### ISSUE THREAD

**Issue by jvsinghk** [jvsinghk/ravenous#5]

**Comment by adgai19** [comment]
Can I work on this?

**Comment by jvsinghk** [comment]
Sure @adgai19. You would need a Yelp API Key for your app let me update the Readme than will assign this to you.

**Comment by jvsinghk** [comment]
@adgai19 I have updated the readme and assigned the issue to you. Let me know if you have any questions. Happy contributing

**Comment by jvsinghk** [comment]
@adgai19 Thanks for your contribution. Feel free to connect with me over LinkedIn or Twitter @jvsinghk handle there too. Happy Hacking!

**Comment by jvsinghk** [comment]
@adgai19 Please share your approach which you folowed to resolve this as well here

**Comment by adgai19** [comment]
> @adgai19 Please share your approach which you folowed to resolve this as well here

Sure. 
The google maps api has a query parameter to search for places in google maps. The documentation for the api is [here](https://developers.google.com/maps/documentation/urls/get-started). I have set the search parameter as the the business name and address and used string replace to replace whitespace with +. This query gives the href for the anchor tag. 
I then wrapped the address element in anchor element and added anchor tag with href property that I got from above. I think that this will cover all if not most cases as business name and address is a pretty unique combination. In cases where that is not the case, the city can be added to the query string.
